package.json

```diff
@@ -55,22 +55,22 @@
   },
   "devDependencies": {
     "@astrojs/check": "^0.9.4",
-    "@biomejs/biome": "2.0.0",
+    "@biomejs/biome": "2.0.6",
     "@changesets/changelog-github": "^0.5.1",
-    "@changesets/cli": "^2.28.1",
-    "@types/node": "^18.19.50",
-    "esbuild": "^0.25.0",
-    "eslint": "^9.24.0",
-    "eslint-plugin-regexp": "^2.7.0",
-    "knip": "5.50.5",
+    "@changesets/cli": "^2.29.5",
+    "@types/node": "^18.19.115",
+    "esbuild": "0.25.5",
```

Comment:
Can you add the reason why we pinned ˋesbuildˋ as the only one, while others only got the version bump?

Reply:
esbuild doesn't follow semver, which means that a minor could break the code. I could change it to `~`, but we shouldn't use `^`